src/pages/CreateFeedback.tsx

```diff
@@ -1,206 +1,105 @@
 import { useState } from 'react';
 import { ProductRequest } from '../types';
-import { Link, useNavigate } from 'react-router-dom';
+import { useNavigate, Link } from 'react-router-dom';
 import arrowDown from '../assets/shared/icon-arrow-down.svg';
 import arrowUp from '../assets/shared/icon-arrow-up.svg';
 import leftArrow from '../assets/shared/icon-arrow-left.svg';
 import newFeedback from '../assets/shared/icon-new-feedback.svg';
-
+import { useForm, SubmitHandler } from 'react-hook-form';
 interface IProps {
   productRequests: ProductRequest[];
 }
 
+type Inputs = {
+  titleRequired: string;
+  feedbackRequired: string;
+  category: string;
+};
+
 export const CreateFeedback = (props: IProps) => {
   const { productRequests } = props;
-  const [isOpen, setIsOpenState] = useState<boolean>(false);
-  const [category, setCategory] = useState<string>('Feature');
-  const [openBorder, setOpenBorder] = useState<string>('');
-  const [title, setTitle] = useState<string>('');
-  const [feedback, setFeedback] = useState<string>('');
-  const [arrow, setArrow] = useState<string>('down arrow');
   const navigate = useNavigate();
 
-  const openDropDown = () => {
-    if (isOpen === true) {
-      setIsOpenState(false);
-      setOpenBorder('');
```

Comment:
Look how much was cleaned up! So awesome. No need for all the `useState` either 👍

Reply:
It's awesome!